examples/flutter_gallery/lib/gallery/icons.dart

```diff
@@ -5,6 +5,9 @@
 import 'package:flutter/material.dart';
 
 class GalleryIcons {
+  // This class is intended to be used as an interface, and should not be
```

Comment:
This class is not really intended to be used an an interface. It's just a grouping mechanism for a collection of statics. Extending (or implementing it) just doesn't make much sense.

Reply:
Same is true for some of the other classes that only have statics below.

Reply:
I believe I fixed all of these. Sorry for the delay.